app/services/github_repository_client.rb

```diff
@@ -0,0 +1,15 @@
+class GithubRepositoryClient
+  URL = 'https://api.github.com/repos/rootstrap'.freeze
```

Comment:
Can we extract the organization name to another constant?

Reply:
is there cases where the org could change?